README.md

```diff
@@ -1,9 +1,14 @@
 # eredis (Nordix fork)
 
+[![Hex pm](http://img.shields.io/hexpm/v/eredis.svg?style=flat)](https://hex.pm/packages/eredis)
+
```

Comment:
Add a "build passing" badge as well?

Reply:
Added a build and downloads badge/shield